content/blog/2021/06/camel-quarkus-release-2.0.0/index.md

```diff
@@ -0,0 +1,92 @@
+Many thanks to all contributors and issue reporters!
+
+## Quarkus 2.0.0.Final
+
+Like in Camel 3.11, in Quarkus 2.0, Vert.x and MicroProfile were upgraded to their newest major versions 4.
+Further, in Quarkus 2.0, Java 8 support was dropped permanently and Java 11 is the minimal supported version.
+21.1 is the recommended version of GraalVM.
+
+When it comes to developer experience, Quarkus 2.0 introduces continuous testing.
+Quarkus dev mode (a.k.a. `mvn quarkus:dev`) exists since the very beginnings of Quarkus.
+It not only starts your application so that you can test it manually,
+but it also keeps watching for changes in your workspace, recompiling the modified classes as needed
+and reloading the running application.
+Now with continuous testing, your tests are executed as you change your application code.
+You have to press `r` to enable it.
+You may want to watch [this video](https://www.youtube.com/watch?v=rUyiTzbezjw) to learn more about the new continuous testing feature.
+
+Please refer to [Quarkus 2.0.0.Final announcement](https://quarkus.io/blog/quarkus-2-0-0-final-released/) for more details.
+
+## Camel 3.11.0
+
+Please check the What's new in [Camel 3.11.0 blog post](/blog/2021/06/Camel311-Whatsnew/) for more information.
+
+## New extensions and promotions to native
+
+New extensions:
+
+* [Avro Jackson](/camel-quarkus/latest/reference/extensions/jackson-avro.html)
+* [AWS Secrets Manager](/camel-quarkus/latest/reference/extensions/aws-secrets-manager.html) (JVM only)
+* [Azure CosmosDB](/camel-quarkus/latest/reference/extensions/azure-cosmosdb.html) (JVM only)
+* [Etcd3](/camel-quarkus/latest/reference/extensions/etcd3.html) (JVM only)
+* [Google Cloud Functions](/camel-quarkus/latest/reference/extensions/google-functions.html) (JVM only)
+* [Google Storage](/camel-quarkus/latest/reference/extensions/google-storage.html)
+* [jOOR](/camel-quarkus/latest/reference/extensions/joor.html) (JVM only)
+* [Kamelet Reify](/camel-quarkus/latest/reference/extensions/kamelet-reify.html) (JVM only)
+* [Protobuf Jackson](/camel-quarkus/latest/reference/extensions/jackson-protobuf.html)
+
+Extensions newly supported in native mode:
+
+* [DigitalOcean](/camel-quarkus/latest/reference/extensions/digitalocean.html)
+* [OpenStack](/camel-quarkus/latest/reference/extensions/openstack.html)
+* [XChange](/camel-quarkus/latest/reference/extensions/xchange.html)
+* [XQuery](/camel-quarkus/latest/reference/extensions/xchange.html)
+
+
+## Test coverage and closing functionality gaps
+
+In this release, we invested a lot of effort into reviewing existing tests and adding new test cases.
+We proceeded methodically, following the main Camel documentation.
+We took care to cover every use case mentioned on the Camel component pages.
+For example for the HTTP component, we went through the use cases mentioned on its component page](/components/latest/http-component.htm),
```

Comment:
Should it be http-component.html ?

Reply:
and there is a missing "[" for this link. Look at the preview

Reply:
+1, thanks!